awsrequests/connections.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+from botocore.exceptions import ClientError
+import boto3
+
+class AWSConnections(object):
+    def __init__(self, **kwargs):
+        self.args = kwargs
+        if 'client' in self.args['resource']:
+            self.client = self.setClient()
```

Comment:
Another thing I don't understand is what this client is used for.  We don't make any AWS API calls in this library beyond the sts-ing, and I don't see it ever used anywhere, only ever assigned.